src/utils/http/handleError.js

```diff
@@ -0,0 +1,64 @@
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import endUserMessages from '../endUserMessages';
+
+const httpStatusCodes = require('./httpStatusCodes');
+
+const handleCodedErrors = (error, message, notifyUser) => {
```

Comment:
this `handleError.js` file looks specific for errors -- why don't you move it under `http/errors`?

Reply:
because `fetchAPI` & `handleError` should be used together so it makes sense for them to be at the same folder level IMO. Moreover, the `/errors` folder only contains actual errors types, not code